packages/@uppy/companion/src/server/middlewares.js

```diff
@@ -42,7 +42,7 @@ exports.verifyToken = (req, res, next) => {
   const { err, payload } = tokenService.verifyEncryptedToken(token, req.companion.options.secret)
   if (err || !payload[providerName]) {
     if (err) {
-      logger.error(err, 'token.verify.error', req.id)
+      logger.error(err.message, 'token.verify.error', req.id)
```

Comment:
why not send the whole err object like in the other instances?

Reply:
because it's a very common error and causes a lot of noise, and in the current version we only log the message (inside logger.error), so I thought we'd keep the noise low on this one